cmd/samples/common/factory.go

```diff
@@ -30,9 +26,9 @@ func NewBuilder(logger *zap.Logger) *WorkflowClientBuilder {
 	}
 }
 
-// SetHostPort sets the hostport for the builder
-func (b *WorkflowClientBuilder) SetHostPort(hostport string) *WorkflowClientBuilder {
-	b.hostPort = hostport
+// SetHostPort sets the host:port for the builder
+func (b *WorkflowClientBuilder) SetHostPort(hostPort string) *WorkflowClientBuilder {
```

Comment:
Do we really need this builder abstraction anymore?

Reply:
No. Entire `common` will go away when I complete all other samples. I refactored only `cron` so far.